content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,242 @@
+      :align: center
+      :alt: Open stock move report
+
+To follow an operation identified from the :guilabel:`Product Moves` report, navigate to
+:menuselection:`Inventory --> Operations --> Transfers`. This is the page that displays all
+warehouse transfers-- internal or external. Search for and click on the corresponding
+:guilabel:`Reference` operation to view the document explaining why the product was moved to the
+wrong location.
+
+.. tip::
+   When there are too many records, use either the :guilabel:`Filters` or :guilabel:`Group by`
+   features to sort and :ref:`find desired records<inventory/putaway/use-advanced-filters>`. It is
+   useful to search exclusively for the receipt operation type because putaway rules store products
+   *into* a specific stock location.
+
+.. example::
+   Continuing to diagnose why the product, `Apple`, was placed in `WH/Stock/Shelf 1`, navigate to
+   the list of all transfers in :menuselection:`Inventory --> Operations --> Transfers`. Click on
+   the list item where the :guilabel:`Reference` field matches the `WH/IN/00015` operation
+   identified in the :ref:`previous example<inventory/putaway/stock_moves>` to open the receipt.
+
+   .. image:: putaway/transfers.png
+      :align: center
+      :alt: Select the desired record
+
+   This receipt describes which products were received from the supplier, with the affiliated
+   :abbr:`PO (Purchase Order)` linked in the :guilabel:`Source Document` field. On the warehouse
+   reception document, the field :guilabel:`Destination Location` shows where the product is
+   received upon entering the warehouse.
+
+   Clicking on the :guilabel:`Detailed Operations` icon ( ≣ ) to the right of the product line opens
+   a window that shows the product was moved to the wrong location. Based on this information it can
+   be gathered that the putaway rule was triggered but the location was set incorrectly. Rectify
+   this in :menuselection:`Inventory --> Configuration --> Putaway Rules`.
+
+   .. image:: putaway/detailed-operations.png
+      :align: center
+      :alt: Click Detailed Operations button on the right of warehouse transfers page
+
+   .. image:: putaway/automated.png
+      :align: center
+      :alt: Show putaway rule in detailed operations popup
+
+.. _inventory/putaway/use-advanced-filters:
+
+Search for products using advanced filters
+------------------------------------------
+
+Keep track of and trace products on the :guilabel:`Product Moves` page. Navigate to the page from
+:menuselection:`Inventory --> Reporting --> Product Moves` and use the :guilabel:`Filters` and
+:guilabel:`Group by` features to stay updated on warehouse operations.
+
+Transfers based on putaway rules can be found by toggling on the :guilabel:`Internal` filter as
+mentioned in this :ref:`tip<inventory/putaway/internal-stock-moves>` previously. Other useful
+filters and groupbys intended for reporting on the company's internal operations include:
```

Comment:
Please give these UI elements a :guilabel:

Reply:
I'm uncertain which UI elements need a guilabel here. Could you please clarify? (:

Reply:
I feel that **Filters** and **Group by** deserve a :guilabel: because we are listing the Filters and Group By options that are available. What do you think?

<img width="156" alt="Screenshot 2023-05-09 at 5 14 25 PM" src="https://github.com/odoo/documentation/assets/36018073/20ffcff0-6381-4c5c-aacb-33b38ac58869">

Reply:
In the section above, guilabels were added to Filters and Group by because I was calling attention to the buttons like you've shown above,

```
Keep track of and trace products on the :guilabel:`Product Moves` page. Navigate to the page from
:menuselection:`Inventory app --> Reporting --> Product Moves` and use the :guilabel:`Filters` and
:guilabel:`Group by` features to stay updated on warehouse operations.

Transfers based on putaway rules can be found by toggling on the :guilabel:`Internal` filter as
mentioned in this :ref:`tip <inventory/routes/strategies/putaway/internal-stock-moves>` previously.
Other useful filters and groupbys intended for reporting on the company's internal operations
include:
```
But in the second section, I'm uncertain about surrounding "filters and groupbys" with a guilabel because I'm referring to different types of filter and groupby functions-- distinct from the UI element.

Reply:
Ok I see where you are coming from and I agree with your reasoning. I'm ok leaving it the way you have it 🙂